lib/zipkin-tracer/tracer_factory.rb

```diff
@@ -11,6 +11,11 @@ def tracer(config)
         when :kafka
           require 'zipkin-tracer/zipkin_kafka_tracer'
           Trace::ZipkinKafkaTracer.new(zookeepers: config.zookeeper)
+        when :kafka_producer
+          require 'zipkin-tracer/zipkin_kafka_tracer'
+          options = { producer: config.kafka_producer }
```

Comment:
you can do 
```
 options = { producer: config.kafka_producer, topic:  config.kafka_topic }
Trace::ZipkinKafkaTracer.new(options)
```
And it will work the same way.